From 987105c1ff12b781c3240f826bb5f9709e987c05 Mon Sep 17 00:00:00 2001
From: William Wu <william.wu@rock-chips.com>
Date: Thu, 12 Jul 2018 08:40:37 +0800
Subject: [PATCH] usb: dwc_otg_310: resize the depth of TxFIFO number 1 and 2

The commit c64eaccc619f ("usb: dwc_otg_310: map ep to
corresponding TxFIFO number") use a fixed mapping between
the endpint addresses and the TxFIFO number. The TxFIFO
number 1 is assigned to ep1-in, and the TxFIFO number 2
is assigned to ep3-in. For uvc function, the ep1-in is
used for uvc control ep (interrupt), and the ep3-in is
used for uvc stream ep. So resize the depth of TxFIFO
number 1 to 512 Bytes, and resize the depth of TxFIFO
number 2 to 1024 Bytes.

Change-Id: I07a89aab01206f6b6f853886d564130abffc1586
Signed-off-by: William Wu <william.wu@rock-chips.com>
---
 drivers/usb/dwc_otg_310/dwc_otg_driver.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/usb/dwc_otg_310/dwc_otg_driver.c b/drivers/usb/dwc_otg_310/dwc_otg_driver.c
index 19a0a06abee4..d309af31e9fe 100644
--- a/drivers/usb/dwc_otg_310/dwc_otg_driver.c
+++ b/drivers/usb/dwc_otg_310/dwc_otg_driver.c
@@ -183,8 +183,8 @@ static struct dwc_otg_driver_module_params dwc_otg_module_params = {
 	.en_multiple_tx_fifo = -1,
 	.dev_tx_fifo_size = {
 			     /* dev_tx_fifo_size */
-			     0x100,
 			     0x80,
+			     0x100,
 			     0x80,
 			     0x60,
 			     0x10,
-- 
2.35.3

